src/Chainweb/BlockHeader.hs

```diff
@@ -373,10 +355,10 @@ powTarget
         -- ^ block creation time of new block
     -> HashTarget
         -- ^ POW target of new block
-powTarget p bct@(BlockCreationTime bt) = case effectiveWindow p of
+powTarget p (BlockCreationTime bt) = case effectiveWindow p of
     Nothing -> maxTarget
     Just w
-        | isLastInEpoch p || slowEpoch p bct ->
+        | isLastInEpoch p ->
```

Comment:
It seems that there has been at least one `slowEpoch` event in the history of `mainnet01`. So we'll have to keep this rule for all block height smaller than some height that is larger than the height of the last `slowEpoch` event.

I propose something like

```haskell
| slowEpochGuard p | isLastInEpoch p || slowEpoch p bct ->
```

With `slowEpochGuard p = _blockHeight p < 60,000`. (The constant should be adapted to a few days in the future from the release of the version that has this change.)

Reply:
There may have been more than one. Once we add the new logic, let's do the catch up again and see if it gets caught anywhere else.